### ISSUE TITLE
civic.json thumbnail link

### ISSUE THREAD

**Issue by stvnrlly** [rebeccamills/whatsamata#2]
Hi there, it looks like the link used for the thumbnail is not an image, so the alt text is shown instead. Do you have one that you could use there? If not, just leave the field blank.

**Comment by rebeccamills** [comment]
Yes, I have a better URL to use for that. Can I edit it right in Github, or
should I create and download a whole new JSON file?

Thanks,
Rebecca

On Mon, Mar 23, 2015 at 10:52 AM, stvnrlly notifications@github.com wrote:

> Hi there, it looks like the link used for the thumbnail is not an image,
> so the alt text is shown instead. Do you have one that you could use there?
> If not, just leave the field blank.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/rebeccamills/whatsamata/issues/2.

## 

Rebecca Mills
www.RebeccaMillsDC.com | (202) 503-9866 | @RebeccaMillsDC

**Comment by stvnrlly** [comment]
Right in GitHub is perfect. Our scraper will check the file on GitHub periodically for any changes that you've made.

(This is the script that we run to check, if you're curious: https://github.com/codefordc/files/blob/master/scripts/civic_json.py)

**Comment by rebeccamills** [comment]
Got it. Thanks! I'm leaving it blank for now, since after looking at the
page I'm realizing that it's mostly suited to logos, and I don't have one
of those (yet!).

On Mon, Mar 23, 2015 at 11:52 AM, stvnrlly notifications@github.com wrote:

> Right in GitHub is perfect. Our scraper will check the file on GitHub
> periodically for any changes that you've made.
> 
> (This is the script that we run to check, if you're curious:
> https://github.com/codefordc/files/blob/master/scripts/civic_json.py)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/rebeccamills/whatsamata/issues/2#issuecomment-85060228
> .

## 

Rebecca Mills
www.RebeccaMillsDC.com | (202) 503-9866 | @RebeccaMillsDC
